### DISCUSSION TITLE
unzip module in SDK home path

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13698] [Q&A] [answered]
After the build completes, I have to [unzip the built module ](https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_How-tos/Extending_Titanium_Mobile/Android_Module_Development_Guide/Android_Module_Quick_Start.html#build-and-package-the-module)in the Titanium SDK home path. Is  `~/.titanium/mobilesdk/linux/11.1.1.GA/module/` the correct path?

**Comment by m1ga** [comment, marked as answer]
yes, if you want to have the modules globally, then this is the correct folder. Be sure to check the content of the ZIP. If it is a normal module the `module` folder is inside the file too, so you place it in the SDK main folder (otherwise you'll end up with `module/module/android/...`)

If you just want to use it in a project: simply place the module in the apps root folder and build it. It will extract itself when you run the build command. https://fromzerotoapp.com/use-modules-in-your-app/#link5

**Reply by beppo-ivel** [reply]
> yes, if you want to have the modules globally

What is the difference between `~/.titanium/mobilesdk/linux/<version>/module/` and `~/.titanium/modules/android/`, the scope?

**Reply by m1ga** [reply]
I could only guess that the `<version>` one will only work if you use the SDK version. But I never used that one (it's empty in my SDK folders). I know that the bundled modules like ti.map or hyperloop are in the `.titanium/modules` to be globally available. But I always prefer the app module folder because then I have all modules at the correct version in my git repos/backups

**Reply by de1mat** [reply]
Agreed with @m1ga. I have tried modules stored in both places and prefer to keep them in the project `modules` folder so they are stored in the git repo alongside the app. Especially if you switch to a new computer in the future and you haven't worked on an app for a few months, you will appreciate having everything kept together.
